presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -71,13 +72,14 @@
     public TestMySqlTypeMapping()
             throws Exception
     {
-        this(new TestingMySqlServer("testuser", "testpass", ImmutableList.of("tpch"), MY_SQL_OPTIONS));
+        this.mysqlServer = new TestingMySqlServer("testuser", "testpass", ImmutableList.of("tpch"), MY_SQL_OPTIONS);
```

Comment:
This is still creating a resource in the constructor. It also happens in 2-3 places above.

Reply:
I'll move it to @BeforeClass

Reply:
I'll move it to @BeforeVlass

Reply:
That won't work because createdQueryRunner is called before @BeforeClass